File: Implementacija/rezervacija/src/main/java/com/rezervacija/services/ProjekcijaService.java
package com.rezervacija.services;

import java.util.List;

import com.rezervacija.models.Projekcija;

public interface ProjekcijaService {

	public Projekcija findOne(Long id);
	public List<Projekcija> find();
	public Projekcija save(Projekcija projekcija);
	public void delete(Projekcija projekcija);
	public void deleteById(Long id);
	public Projekcija update(Projekcija projekcija, Long id);
	public String checkBioskop(Long idBioskopa);
	public String checkFilm(Long idFilma);
	public String checkSala(Long idBioskopa, Long idSale);
	public int getBrojZauzetihMesta(Long idProjekcije, int red);
	public int getBrojZauzetihMestaIzmena(Long rezervacijaId, Long projekcijaId, int red);
}

File: Implementacija/rezervacija/src/main/resources/static/js/services/projekcijaService.js
angular.module('rezervacijaApp.ProjekcijaService', [])
	.factory('ProjekcijaService', function($http) {
		
		var factory = {};
		
		factory.getBrojZauzetihMestaPoRedu = function(projekcijaId, red) {
			return $http.get('/rezervacija-service/projekcija/zauzeta/'+projekcijaId+'/'+red);	
		}
		
		factory.getBrojZauzetihMestaPoReduIzmena = function(rezervacijaId, projekcijaId, red) {
			return $http.get('/rezervacija-service/projekcija/zauzetaIzmena/'
					+rezervacijaId+'/'+projekcijaId+'/'+red);	
		}
	
		return factory;
		
	});